cime_config/allactive/config_pesall.xml

```diff
@@ -2660,4 +2660,106 @@
       </pes>
     </mach>
   </grid>
+  <grid name="a%ne30np4.pg2_l%.+_oi%EC30to60E2r2.+w%wQU225EC30to60E2r2">
+    <mach name="cori">
+      <pes compset=".*EAM.+ELM.+MPASSI.+MPASO.+MOSART.+WW3." pesize="M">
+        <comment> --compset WCYCL* --res ne30pg2_EC30to60E2r2_wQU225EC30to60E2r2 on 48 nodes pure-MPI, ~8.8 sypd </comment>
+        <ntasks>
+          <ntasks_atm>5400</ntasks_atm>
+          <ntasks_lnd>1152</ntasks_lnd>
+          <ntasks_rof>1152</ntasks_rof>
+          <ntasks_ice>4320</ntasks_ice>
+          <ntasks_ocn>1440</ntasks_ocn>
+          <ntasks_cpl>4320</ntasks_cpl>
+          <ntasks_wav>2448</ntasks_wav>
```

Comment:
Are these PE layouts just placeholders for now? Just wondering since they seem pretty large for the other components considering that waves is a likely bottleneck. I don't think waves can go above nFreqxnDir for the CD decomposition. That's something we may need to think a little more on since the max number of tasks changes with spectral resolution.

Reply:
yes these values are all just placeholders ... I have no idea what is appropriate for the various machines...

Reply:
Me either, sounds good. I don't know for sure, the E3SM integrators may want to leave this out  in this PR and add verified values in a separate PR. We'll have to see how they want to handle it.